test/built-ins/Temporal/ZonedDateTime/from/argument-propertybag-required-correctly-spelled-properties.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.zoneddatetime.from
+description: Options must contain at least the required correctly-spelled properties.
+features: [Temporal]
+---*/
+
+// object must contain at least the required correctly-spelled properties
+assert.throws(TypeError, () => Temporal.ZonedDateTime.from({
+  years: 1976,
+  months: 11,
+  days: 18,
+  timeZone: "+01:00"
+}));
```

Comment:
This should probably be expanded to be a bit more rigorous - e.g., three tests, each with only one of `year`/`month`/`day` pluralized.

Reply:
Done, though I assume you meant "singular" there rather than "pluralized"?

Reply:
I think I really meant "pluralized" - although now I'm confusing myself. `year`, `month`, `day`, `timeZone` are the required property names for ZonedDateTime property bags, and my suggestion was to have three tests, one with `year` replaced by `years`, and so forth.

Reply:
Ah, got it. Should be done in [0f7dd54](https://github.com/tc39/test262/pull/4291/commits/0f7dd543fa79916117feea7b41da862d46482752)